core/src/components/textarea/textarea.ionic.scss

```diff
@@ -0,0 +1,80 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./textarea.common";
+@import "./textarea.ionic.vars";
+@import "./textarea.ionic.outline.scss";
+
+// Ionic Textarea
+// --------------------------------------------------
+
+:host {
+  --border-radius: #{globals.$ionic-border-radius-400};
+  --color: #{globals.$ionic-color-neutral-1200};
+  --highlight-color-valid: #{globals.$ionic-color-success-base};
+  --highlight-color-invalid: #{globals.$ionic-color-danger-base};
```

Comment:
We should add `ionic-md` to the textarea highlight tests so we have snapshots for valid and invalid.

Reply:
I think we should wait to add the snapshots until we’ve implemented these properly, as they are currently just copied from input which is outdated.